www/chromedriver/Portfile

```diff
@@ -87,7 +82,7 @@ destroot {
 
 # use these to specify python versions, python3 required
 # use ${prefix}/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/_resources/port1.0/group/python-1.0.tcl
-set python_version  310
+set python_version  312
```

Comment:
why can this not use Python 3.13 - that's MacPorts default versions

Reply:
There's some upstream `selenium` issue that breaks here because the `websocket` module doesn't exist in that version.